index.d.ts

```diff
@@ -0,0 +1,25 @@
+import LRU = require("lru-cache");
+
+declare interface AppOptions {
+  id: string
+  privateKey: string
+  baseUrl?: string
+  cache?: LRU.Cache<string, string>
+}
+declare interface getJWTOptions {
+  installation_id: string
+}
+
+declare interface getInstallationAccessTokenOptions {
+  installation_id: string
+}
+
+// Not really a class, but it is how they say it should be used in the readme.
+// In TypeScript, you cannot use the `new` keyword on functions (excluding old-style classes using functions and prototype), only on classes
+declare class App {
+  constructor(options: AppOptions)
+  getSignedJsonWebToken(options?: getJWTOptions): string
```

Comment:
`getSignedJsonWebToken` has no options.

```suggestion
  getSignedJsonWebToken(): string
```

Reply:
and should the methods be prefixed with `public `? Compare https://github.com/probot/probot/blob/e1d78738aa4791940a5161aed2364e42dbc1ce6d/src/%40types/%40octokit/app/index.d.ts